README.md

```diff
@@ -42,6 +42,7 @@ This App allow users book reservations with their mentors.
 - [ ] **Book reservation**
 - [ ] **Show their reservations**
 - [ ] **Delete their reservations**
+- [ ] **Admin features**
```

Comment:
> hi @Meltrust , You can register an account in the front-end, and on the back-end, you can run `rails console` and then find the user that you signed up. example: `user = User.find_by(email: "email@email.com"` Then you can set `user.isAdmin = true` and `user.save` Comeback front-end, now you can login with admin features

Thank you for your kind response.  Using the terminal to give my own user admin privileges is how I did the original review.  

However, 

- [ ] OPTIONAL: these instructions about how to create and test an admin user, should be added to the readme so your users can easily test the admin user, just by following the exact same instructions you just gave me.

Remember, we do this project not just for reviewers but also for potential employers.  We want to make this app an easy and smooth experience for them, running it correctly just by following the readme instructions.

The admin user works really well when accessed.  We want them to come and test it without much effort.  That way, your chances are improved.

Of course, this is optional.  Congratulations and see you in the software industry. 💪

Reply:
Hi @Meltrust,
Thanks so much for your suggestion. It's a valuable advice. I will definitely implement it.
Kind regards,